ios/FluentUI/Drawer/DrawerController.swift

```diff
@@ -123,13 +110,23 @@ open class DrawerController: UIViewController {
     }
 
     /// Set `backgroundColor` to customize background color of the drawer
-    @objc open var backgroundColor: UIColor = Colors.Drawer.background {
+    @objc open lazy var backgroundColor: UIColor = drawerBackgroundColor(fluentTheme: view.fluentTheme) {
         didSet {
             useCustomBackgroundColor = true
             view.backgroundColor = backgroundColor
```

Comment:
A check for `viewIsLoaded` should be added before accessing the view, similar to other didSets in this file.

```
if viewIsLoaded {
     view.backgroundColor = backgroundColor
}
```

Accessing the `view` before is loaded will force a `loadView()`, which will call `viewDidLoad()` and a custom `presentationStyle` will be ignored if the other of setting the properties are not right.

Issue filed here: https://github.com/microsoft/fluentui-apple/issues/1338